htdocs/comm/mailing/card.php

```diff
@@ -677,6 +677,7 @@
 $form = new Form($db);
 $htmlother = new FormOther($db);
 
+$object->fetch($id);
```

Comment:
Because the fetch is already done at begining, it would be better to make this fetch only in the case of we need a refresh/reload.